### PR TITLE
Add docs - admin service accounts 

### PR DESCRIPTION
**Pull request by IrinaSouth**

### CODE REVIEW

File: docs/modules/ROOT/pages/identity.adoc [hunk 1]
```diff
@@ -246,32 +242,29 @@ The two certificates need to be provided to FeatureHub without their surrounding
...
 
-Your SAML provider will ask you what extra attributes are required, so:
+Your SAML provider will ask you what extra attributes are required:
 
  - Last Name = `urn:oid:2.5.4.4`
  - First Name = `urn:oid:2.5.4.42`
- - Display Name = `urn:oid:2.16.840.1.113730.3.1.241` 
-
-And yes, we realise these are bizarre names but they are from the days of OASIS committees where people smarter than
-us thought this was a good idea. We will reject the login if the name isn't available.
```
Comment (rvowles) [inline docs/modules/ROOT/pages/identity.adoc:274]: You need to make it clear that either LastName/FirstName or DisplayName are required and that we will reject the login if the name isn't available (you deleted that bit)

File: docs/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -288,21 +288,24 @@ FeatureHub was designed to ensure that security was built into the platform from
 === Key Security concepts
 
 ==== Administrators
-There are two types of administrators, *Site Administrators* and *Portfolio Administrators*.
+There are two types of administrators, *Organisation Super Admin* and *Portfolio Admin*.
 
 ===== Site Administrators
```
Comment (rvowles) [inline docs/modules/ROOT/pages/index.adoc:293]: You changed the name to "Organisation Super Admin" but didn't change the title here. 🍷

File: docs/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -288,21 +288,24 @@ FeatureHub was designed to ensure that security was built into the platform from
...
-** Create applications
-** Manage access to applications
-** Create Service Accounts
+** Create and manage applications
+** Create and manage environments
+** Manage users and groups access to applications
+** Create service accounts
 
-NOTE: Every Portfolio automatically gets a group called "Administrators", Simply adding people to this group
+NOTE: Every Portfolio automatically gets a group called "Administrators", simply adding people to this group
```
Comment (rvowles) [inline docs/modules/ROOT/pages/index.adoc:308]: Should this be "simply add people to this group"?

**Review by rvowles — CHANGES_REQUESTED** [review]
One important change, one 🍷 change 😉

**Review by rvowles — APPROVED** [review]

File: docs/modules/ROOT/pages/identity.adoc (hunk 1 above)
Comment (IrinaSouth) [reply docs/modules/ROOT/pages/identity.adoc:274]: it already says this ```Either the display name *or* the First Name/Last Name
must be provided, there are no other requirements.```

should be enough?

**Pull request merged**